components/project-card/project-card-detailed.jsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import ReactGA from 'react-ga';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import moment from 'moment';
+import Creators from './meta/creators.jsx';
+import Thumbnail from './meta/thumbnail.jsx';
+import Title from './meta/title.jsx';
+import Description from './meta/description.jsx';
+import WhyInteresting from './meta/why-interesting.jsx';
+import IssuesAndTags from './meta/issues-and-tags.jsx';
+import GetInvolved from './meta/get-involved.jsx';
+import BookmarkControl from '../bookmark-control.jsx';
+import bookmarkManager from '../../js/bookmarks-manager';
+import user from '../../js/app-user.js';
+
+class DetailedProjectCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bookmarked: false
+    };
+  }
+
+  sendGaEvent(category = ``, action = ``, transport = ``) {
+    let config = {
+      category: `Entry Card - ${category}`,
+      action: action,
+      label: `${this.props.id} - ${this.props.title}`
+    };
+
+    if (transport) {
+      config.transport = transport;
+    }
+
+    ReactGA.event(config);
+  }
+
+  componentDidMount() {
+    this.setInitialBookmarkedStatus();
+  }
+
+  setInitialBookmarkedStatus() {
+    if (user.loggedin) {
+      this.setState({
+        bookmarked: this.props.isBookmarked
+      });
+
+      return;
+    }
+
+    let bookmarks = bookmarkManager.bookmarks.get();
+    let bookmarked;
```

Comment:
L57 is already conditional on `bookmarks` so just remove this line and on L56 say `let bookmarked = ...`